updating/preparing_for_updates/updating-cluster-prepare-past-4.18.adoc

```diff
@@ -0,0 +1,123 @@
+
+You can reschedule pods running on {op-system-base} compute nodes to run on your {op-system} nodes instead by using node selectors.
+
+For example, the following `Node` object has a label for its operating system information, in this case {op-system}:
++
+.Sample `Node` object with {op-system} label
+[source,yaml,subs="+attributes"]
+----
+kind: Node
+apiVersion: v1
+metadata:
+  name: ip-10-0-131-14.ec2.internal
+  selfLink: /api/v1/nodes/ip-10-0-131-14.ec2.internal
+  uid: 7bc2580a-8b8e-11e9-8e01-021ab4174c74
+  resourceVersion: '478704'
+  creationTimestamp: '2019-06-10T14:46:08Z'
+  labels:
+    kubernetes.io/os: linux
+    failure-domain.beta.kubernetes.io/zone: us-east-1a
+    node.openshift.io/os_version: '{product-version}'
+    node-role.kubernetes.io/worker: ''
+    failure-domain.beta.kubernetes.io/region: us-east-1
+    node.openshift.io/os_id: rhcos <1>
+    beta.kubernetes.io/instance-type: m4.large
+    kubernetes.io/hostname: ip-10-0-131-14
+    beta.kubernetes.io/arch: amd64
+#...
+----
+<1> The label identifying the OS running on the node, to match the pod node selector.
+
+Any pods that you want to schedule to new {op-system} nodes must contain a matching label in its `nodeSelector` field. The following procedure describes how to add it.
+
+.Procedure
+
+. Deschedule the {op-system-base} node currently running your existing pods:
++
+[source,terminal]
+----
+$ oc adm cordon <rhel-node>
+----
+
+. Add an `rhcos` node selector to a pod:
+
+** To add the node selector to existing and future pods, add the node selector to the controller object for the pods:
++
+.Example `Deployment` object with `rhcos` label
+[source,terminal]
+----
+$ oc patch dc <my-app> -p '{"spec":{"template":{"spec":{"nodeSelector":{"node.openshift.io/os_id":"rhcos"}}}}}'
+----
++
+Any existing pods under your `Deployment` controlling object will be recreated on your {op-system} nodes.
+
+** To add the node selector to a specific, new pod, add the selector to the `Pod` object directly:
++
+.Example `Pod` object with `rhcos` label
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: <my-app>
+#...
+spec:
+  nodeSelector:
+    node.openshift.io/os_id: rhcos
+#...
+----
++
+The new pod will be created on {op-system} nodes, assuming the pod also has a controlling object.
+
+[id="identifying-and-removing-rhel-worker-nodes_{context}"]
+== Identifying and removing {op-system-base} worker nodes
+
+With the introduction of {product-title} 4.19, package-based {op-system-base} worker nodes are no longer supported. The following procedure describes how to identify {op-system-base} nodes for cluster removal on bare-metal installations. You must perform the steps described in this procedure in order to ensure a successful cluster update.
+
+.Procedure
+
+. Identify nodes in your cluster that are running {op-system-base} by running the following command:
```

Comment:
```suggestion
. Identify nodes in your cluster that are running {op-system-base} by entering the following command:
```
Maybe use "entering" here to avoid having two instances of "running" near each other.